tests/resolution/evaluation/test_evaluate_docker_version.py

```diff
@@ -1,28 +1,71 @@
-"""Test evaluation base."""
-# pylint: disable=import-error,protected-access
+"""Test evaluation of docker version."""
+
+import random
 from unittest.mock import patch
 
+# pylint: disable=import-error,protected-access
+from awesomeversion import AwesomeVersion
+
 from supervisor.const import CoreState
 from supervisor.coresys import CoreSys
 from supervisor.resolution.evaluations.docker_version import EvaluateDockerVersion
 
 
-async def test_evaluation(coresys: CoreSys):
-    """Test evaluation."""
+async def test_evaluation_supported(coresys: CoreSys):
+    """Test real evaluation with a current docker daemon."""
```

Comment:
```suggestion
    """Test evaluation with a known good docker version."""
```

I kinda thought at one point it would be good to just let it hit the real thing, but I could not finagle getting pytest not to mock it out.